cross/diesel/Makefile

```diff
@@ -0,0 +1,28 @@
+PKG_NAME = diesel
+PKG_VERS = 2.1.4
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/diesel-rs/diesel/archive
+PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS += cross/mysql-connector-c
+
+# powerpc archs (except qoriq) are not supported
+UNSUPPORTED_ARCHS += $(OLD_PPC_ARCHS)
+
+HOMEPAGE = https://diesel.rs/
+COMMENT  = A safe, extensible ORM and Query Builder for Rust.
+LICENSE  = MIT
+
+# build the cli tool for mysql
+RUST_SRC_DIR = $(WORK_DIR)/$(PKG_DIR)/diesel_cli
+CARGO_BUILD_ARGS += --no-default-features
+CARGO_BUILD_ARGS += --features=mysql
+
+# mysqlclient-sys: to find libmysqlclient
+ENV += MYSQLCLIENT_LIB_DIR=$(STAGING_INSTALL_PREFIX)/lib
+# to find dependencies of libmysqlclient (libz)
+ENV += RUSTFLAGS="-Clink-arg=-Wl,--rpath-link,$(STAGING_INSTALL_PREFIX)/lib -Clink-arg=-Wl,--rpath-link,$(STAGING_INSTALL_PREFIX)/lib"
```

Comment:
Hey @hgy59, it's advisable to avoid relying on `RUSTFLAGS` unless absolutely essential due to their potential lack of backward compatibility. In this instance, the syntax appears to be incorrect and there seems to be duplication of arguments. Additionally, after reviewing the build logs, it doesn't appear to be invoked. Could you confirm whether it's necessary to include this?